Source/ASSectionController.h

```diff
@@ -35,6 +35,19 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (ASCellNodeBlock)nodeBlockForItemAtIndex:(NSInteger)index;
 
+/**
+ * Similar to -collectionView:cellForItemAtIndexPath:.
+ *
+ * Note: only called if nodeBlockForItemAtIndex: returns nil.
+ *
+ * @param index The index of the item.
+ *
+ * @return A node to display for the given item. This will be called on the main thread and should
+ *   not implement reuse (it will be called once per item).  Unlike UICollectionView's version,
+ *   this method is not called when the item is about to display.
+ */
+- (ASCellNode *)nodeForItemAtIndex:(NSInteger)index;
```

Comment:
Can't find the implementation of this method.

Reply:
`ASSectionController` is a protocol, so it's for clients to implement it. It's called within `collectionNode:nodeForItemAtIndexPath:`

Reply:
lol, I'm stupid!